widget/charts/src/components/BarChart/BarChart.helpers.ts

```diff
@@ -0,0 +1,76 @@
+import type { BarStackDataType } from './BarChart.types';
+
+import { buildChartTheme } from '@visx/xychart';
+
+export const transactionTheme = buildChartTheme({
+  backgroundColor: 'transparent',
+  colors: ['#469BF5', '#F17606'],
+  gridColor: '#469BF5',
+  tickLength: 8,
+  gridColorDark: '',
+});
+
+export const volumeTheme = buildChartTheme({
+  backgroundColor: 'transparent',
+  colors: ['#F17606', '#469BF5'],
+  gridColor: '#469BF5',
+  tickLength: 8,
+  gridColorDark: '',
+});
+
+export const bottomAxisLayout = {
+  mobile: {
```

Comment:
The logic of the days should not be included in the chart component.
We should add new props for bottomAxisLayout.

Reply:
fixed